draft-thoji-scone-trone-protocol.md

```diff
@@ -186,9 +186,16 @@ signaling occurs as this is specific to the application in use.
 A signal does not prove that a higher rate would not be successful.  Endpoints
 that receive this signal therefore need to treat the information as advisory.
 
-As an advisory signal, network elements cannot assume that endpoints will
-respect the signal.  Though this might reduce the need for more active rate
-limiting, how rate limit enforcement is applied is a matter for network policy.
+The fact that an endpoint requests bitrate signals does not necessarily mean
+that it will adhere to them; in some cases, the endpoint cannot. In certain
+situations, a flow may initially be used to serve video chunks, with the client
```

Comment:
```suggestion
that it will adhere to them; in some cases, the endpoint cannot. For
example, a flow may initially be used to serve video chunks, with the client
```